test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/Export/CosmosDbExportTests.cs

```diff
@@ -0,0 +1,174 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.Models;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.CosmosDb.Features.Operations.Export;
+using Microsoft.Health.Fhir.CosmosDb.Features.Storage;
+using Microsoft.Health.Fhir.CosmosDb.Features.Storage.Operations;
+using Microsoft.Health.Fhir.CosmosDb.Features.Storage.Queues;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.JobManagement;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Health.Fhir.Tests.Integration.Persistence
+{
+    [Trait(Traits.OwningTeam, OwningTeam.Fhir)]
+    [Trait(Traits.Category, Categories.DataSourceValidation)]
+    public class CosmosDbExportTests : IClassFixture<CosmosDbFhirStorageTestsFixture>
+    {
+        private readonly CosmosDbFhirStorageTestsFixture _fixture;
+        private readonly ITestOutputHelper _testOutputHelper;
+        private readonly ISearchService _searchService;
+        private readonly CosmosFhirOperationDataStore _operationDataStore;
+        private readonly IQueueClient _queueClient;
+        private readonly IFhirStorageTestHelper _fhirStorageTestHelper;
+        private readonly byte _queueType = (byte)QueueType.Export;
+
+        public CosmosDbExportTests(CosmosDbFhirStorageTestsFixture fixture, ITestOutputHelper testOutputHelper)
+        {
+            _fixture = fixture;
+            _testOutputHelper = testOutputHelper;
+            _searchService = _fixture.GetService<ISearchService>();
+            _operationDataStore = _fixture.GetService<CosmosFhirOperationDataStore>();
+            _queueClient = _fixture.GetService<IQueueClient>();
+            _fhirStorageTestHelper = _fixture.GetService<IFhirStorageTestHelper>();
+        }
+
+        [Fact]
+        public async Task ExportWorkRegistration()
```

Comment:
Should this be one test? It looks like it is three different tests.

Reply:
Will investigate - I used the same format as the SQL tests but agree it doesn't seem right.